addons/survey/views/survey_question_views.xml

```diff
@@ -97,9 +103,9 @@
                         </group>
                     </group>
                     <notebook>
-                        <page string="Answers" name="answers" attrs="{'invisible': [('is_page', '=', True)]}">
+                        <page string="Answers" name="answers" attrs="{'invisible': ['|', ('is_page', '=', True), ('question_type', '=', 'text_box')]}">
                             <div class="row">
-                                <div class="col-6"
+                                <div class="col-6" 
```

Comment:
Extra end-of-line space here.
(There are some IDE extensions that can help with that)